pkg/services/authz/zanzana/schema/dashboard.fga

```diff
@@ -29,16 +29,20 @@ type dashboard
   relations
     define org: [org]
 
-    define read: [user, team#member, role#assignee] or dashboard_read from org
-    define write: [user, team#member, role#assignee] or dashboard_write from org
-    define delete: [user, team#member, role#assignee] or dashboard_delete from org
+    # Action sets
+    define view: [user, team#member, role#assignee]
+    define edit: [user, team#member, role#assignee]
+    define admin: [user, team#member, role#assignee]
+
     define create: [user, team#member, role#assignee] or dashboard_create from org
-    define permissions_read: [user, team#member, role#assignee] or dashboard_permissions_read from org
-    define permissions_write: [user, team#member, role#assignee] or dashboard_permissions_write from org
+    define read: [user, team#member, role#assignee] or view or edit or admin or dashboard_read from org
```

Comment:
How are we defining inheritance here? `dashboard_read` from folder would also grant `read`, right?

Reply:
We're decided to move inheritance logic into the implementation. So we basically perform 2 checks - one for resource itself and one for the parent folder and then return combined answer. So there's no inheritance anymore in schema (except folder-to-folder).